docs/content/components/skipnav.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Skipnav Component - NCI Design System
+---
+
+import { Tabs, TabItem } from '../../src/components/Tabs';
+
```

Comment:
There was guidance provided in the original ticket.  
https://github.com/NCIOCPL/ncids/issues/317#issuecomment-1024615096